jobs/build/ocp/Jenkinsfile

```diff
@@ -376,7 +376,7 @@ node(TARGET_NODE) {
                 buildlib.set_rpm_spec_version( "origin.spec", NEW_VERSION )
                 buildlib.set_rpm_spec_release_prefix( "origin.spec", NEW_RELEASE )
                 // Note that I did not use --use-release because it did not maintain variables like %{?dist}
-                sh "tito tag --accept-auto-changelog --keep-version --debug"
+                sh 'tito tag --accept-auto-changelog --keep-version --debug --changelog="Automatic commit of package [atomic-openshift] release [${NEW_VERSION}-${NEW_RELEASE}]; bump origin-web-console ${VC_COMMIT}"'
```

Comment:
' in groovy means don't do any string substitution for ${vars}, so your ${NEW_VERSION} etc are literal values here. Change to:
```
sh "tito tag --accept-auto-changelog --keep-version --debug --changelog='Automatic commit of package [atomic-openshift] release [${NEW_VERSION}-${NEW_RELEASE}]; bump origin-web-console ${VC_COMMIT}' "
```

Groovy will do the string substitution and ultimately pass bash a fully formed string in single quotes for the --changelog parameter.